keyboards/kyria/keymaps/default/keymap.c

```diff
@@ -0,0 +1,244 @@
+/* Copyright 2019 Thomas Baart <thomas@splitkb.com>
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include QMK_KEYBOARD_H
+
+#define _QWERTY 0
+#define _LOWER 1
+#define _RAISE 2
+#define _ADJUST 3
+
+enum custom_keycodes {
+  LOWER = SAFE_RANGE,
+  RAISE
+};
```

Comment:
You can replace these with `MO`, if you add:
https://docs.qmk.fm/#/ref_functions?id=update_tri_layer_statestate-x-y-z

```c
layer_state_t layer_state_set_user(layer_state_t state) {
  return update_tri_layer_state(state, _LOWER, _RAISE, _ADJUST);
}
```

And this would allow the use of LT and other layer functions to use the tri layer stuff.   The caveat is that you won't be able to access the adjust layer without them (eg `MO(_ADJUST)` won't work)